tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -2451,6 +2507,20 @@ exports[`json.test.w 1`] = `
    |                          ^ Json literal must have an element
 
 
+error: Unexpected 'array_literal'
+   --> ../../../examples/tests/invalid/json.test.w:62:11
+   |  
+62 |     \\"1\\" => {[ 
+   | /-----------^
+63 | |     {
+64 | |       foo: \\"bar\\",
+65 | |       stuff: [],
+66 | |     }
+67 | |   //^ Missing required field \\"maybe\\" from \\"StructyJson\\"
+68 | |   ]}
+   | \\\\---^ Unexpected 'array_literal'
```

Comment:
Looks like a regression. The line in that file probably needs to be changed to this

```wing
let notJsonMissingFieldArray = {
  "1" => Set<Array<StructyJson>> [[ 
    {
      foo: "bar",
      stuff: [],
    }
  //^ Missing required field "maybe" from "StructyJson"
  ]]
};
```

Reply:
oh my, how many sets there were